### DISCUSSION TITLE
Sonoff Pow R3 reset energy total via mqtt

### DISCUSSION THREAD

**Discussion by ombonbon** [arendst/Tasmota#17515] [Q&A] [answered]
Hallo,

As the subject says, I would like to know if it's possible to reset energy total value of Sonoff Pow R3 via mqtt. I know I can reset it via web GUI's console.
If it's not possible, is there any other way to reset it? I tried HTTP request but always get the energy total value instead of resetting the value.

BR,

**Comment by sfromis** [comment, marked as answer]
Any command available via the web UI console is also available via MQTT (preferred stratgy), serial and http requests as long as you make correct URL escapes, and whatever needed from the place you do it, like not being caught up by shell parsing

**Comment by ombonbon** [comment]
Thanks @sfromis for your answer. I tried to publish value 0 to %topic%/energytotal and I see the energy total calculation is reset. I was confused by the value of energy total because it's showing the same value as energy today but then I see the total start time is changed (which means the energy total calculation has been reset).
Thanks again! :)

**Reply by sfromis** [reply]
What happens is that setting `EnergyTotal` resets the midnight value, but then the result will show the "now" value including consumption recorded today, otherwise it would be showing the wrong number.
